deploy/helm/templates/logs-daemon-set.yaml

```diff
@@ -36,6 +36,38 @@ spec:
       tolerations:
         {{ toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:
+        - name: otel-endpoint-check
+          image: fullstorydev/grpcurl
+          command: ['/bin/grpcurl', '-expand-headers',
+                    '-proto', 'opentelemetry/proto/collector/logs/v1/logs_service.proto',
+                    '-H', 'Authorization: Bearer ${SOLARWINDS_API_TOKEN}',
+                    '-d', '{ "resourceLogs": [ { "resource": { "attributes": [ { "key": "sw.k8s.log.type", "value": { "stringValue": "journal" } } ] } } ] }',
```

Comment:
can you add `sw.k8s.cluster.uid` resource attribute, so that the Log is assigned to the cluster and searchable through log explorer?

And what about adding there some log message?

Reply:
Log message modified.